src/main/java/uk/gov/justice/hmpps/prison/api/model/AssessmentDetail.java

```diff
@@ -61,6 +69,21 @@ public AssessmentDetail(final AssessmentSummary summary, final String assessment
     @Schema(description = "The reason for the review of the classification", example = "HI")
     private String classificationReviewReason;
 
+    @Schema(description = "Overriding classification code", example = "HI")
+    private String overridingClassificationCode;
+
+    @Schema(description = "Originally calculated classification code", example = "HI")
```

Comment:
this feels confusing.  This is the originally calculated classification code, which is different from the `originalClassificationCode`.  I don't know enough about the CSRA process to know what the better description could be though. Does the calculation happen after review - I noticed that `reviewedClassification` is not being returned here.  Would be good to include more information in the descriptions to help people understand what is going on.

Reply:
Yeah it's all a bit confusing - The `originalClassificationCode` only gets populated when something has been approved and it was different to the one that was approved and is used in the CSRA Summaries as well. That can be useful for seeing at a glance if it was changed, but for a more detailed CSRA view you'd want to see both the original and approved specifically (hence the adding of the field).

I can update the description for them to the best of my knowledge though yeah - for reference, the `calculatedClassificationCode` is one that's automatically calculated by the system based on the questions answered in the initial review